<ui version="4.0" >
 <class>QuickConnect</class>
 <widget class="QDialog" name="QuickConnect" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Quick Connect</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_1" >
     <property name="title" >
      <string>&amp;Login</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="editPassword" >
        <property name="echoMode" >
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="textLabel_2" >
        <property name="text" >
         <string>&amp;Password:</string>
        </property>
        <property name="buddy" >
         <cstring>editPassword</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="editUsername" />
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="textLabel_1" >
        <property name="text" >
         <string>&amp;Username:</string>
        </property>
        <property name="buddy" >
         <cstring>editUsername</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2" >
       <widget class="QCheckBox" name="chkStore" >
        <property name="text" >
         <string>&amp;Store password on this computer</string>
        </property>
        <property name="shortcut" >
         <string>Alt+S</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Ser&amp;ver:</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="editHost" />
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="textLabel_4" >
        <property name="text" >
         <string>ICS p&amp;ort:</string>
        </property>
        <property name="buddy" >
         <cstring>spinPort</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QSpinBox" name="spinPort" >
        <property name="maximum" >
         <number>65536</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="textLabel_3" >
        <property name="text" >
         <string>ICS &amp;host:</string>
        </property>
        <property name="buddy" >
         <cstring>editHost</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupTimeseal" >
     <property name="focusPolicy" >
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="title" >
      <string>&amp;Timeseal</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="2" >
       <widget class="QToolButton" name="buttonTimesealPath" >
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="editTimeseal" />
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="textLabel_5" >
        <property name="text" >
         <string>&amp;File Path:</string>
        </property>
        <property name="buddy" >
         <cstring>editTimeseal</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3" >
       <widget class="QLineEdit" name="editTimesealCmd" >
        <property name="toolTip" >
         <string/>
        </property>
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3" >
       <widget class="QCheckBox" name="chkTimesealCmd" >
        <property name="text" >
         <string>Use &amp;custom command line arguments</string>
        </property>
        <property name="shortcut" >
         <string>Alt+S</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>327</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
       <property name="shortcut" >
        <string/>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut" >
        <string/>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>editUsername</tabstop>
  <tabstop>editPassword</tabstop>
  <tabstop>chkStore</tabstop>
  <tabstop>editHost</tabstop>
  <tabstop>spinPort</tabstop>
  <tabstop>groupTimeseal</tabstop>
  <tabstop>editTimeseal</tabstop>
  <tabstop>buttonTimesealPath</tabstop>
  <tabstop>chkTimesealCmd</tabstop>
  <tabstop>editTimesealCmd</tabstop>
  <tabstop>buttonOk</tabstop>
  <tabstop>buttonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>QuickConnect</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>303</x>
     <y>506</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>QuickConnect</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>217</x>
     <y>506</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkTimesealCmd</sender>
   <signal>toggled(bool)</signal>
   <receiver>editTimesealCmd</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>84</x>
     <y>382</y>
    </hint>
    <hint type="destinationlabel" >
     <x>85</x>
     <y>406</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
